modules/ROOT/pages/viewing-api-analytics.adoc

```diff
@@ -17,11 +17,11 @@ When you access the analytics for your APIs, by default you land on your Overvie
 * *Requests by Application*: Bar chart that shows the number of requests from each of the top five registered applications.
 * *Requests by Platform*: Ring chart that shows the number of requests broken down by platform.
 
-image::overviewdash1.png[]
+image::overviewdash1.png["Dashboard overview showing line chart for requests by date, map for requests by location, bar chart for requests by application, and ring chart for requests by platform"]
 
 By default, these charts display the data for APIs in your organization for the past day. You can use the filters at the top left of the page to change the date range or filter to particular APIs. The time ranges reflect your local time zone.
 
-image::filters.png[]
+image::filters.png["Filter controls for date range and API selection in the upper left of the analytics dashboard"]
```

Comment:
Upper left? If you leave it, should it be upper-left?

Reply:
removed